crates/aptos/src/common/types.rs

```diff
@@ -374,8 +375,19 @@ impl CliConfig {
         let aptos_folder = Self::aptos_folder(ConfigSearchMode::CurrentDir)?;
 
         // Create if it doesn't exist
+        let no_dir = !aptos_folder.exists();
         create_dir_if_not_exist(aptos_folder.as_path())?;
 
+        // If the `.aptos/` doesn't exist, we'll add a .gitignore in it to ignore the config file
```

Comment:
We should just ignore .aptos wholesale right? The only other thing that would probably be in there is .aptos/testnet, which they also don't want to upload.

Reply:
There's 2 gitignores for more coverage.  One in the move repo, and one in the .aptos.  I'm not sure what happens if you add a * in .aptos gitignore